go/vt/sqlparser/ast.go

```diff
@@ -558,7 +558,9 @@ type (
 	}
 
 	// Begin represents a Begin statement.
-	Begin struct{}
+	Begin struct {
+		TxCharacteristics []string
```

Comment:
For other, similar things in the AST, we have turned the strings into an enum instead of passing around raw strings. I like these enums - I think it improves the readability and discoverability of the code

Reply:
done